onnxruntime/core/providers/cpu/tensor/squeeze.h

```diff
@@ -13,12 +13,16 @@ class SqueezeBase {
  protected:
   explicit SqueezeBase(const OpKernelInfo& info) {
     std::vector<int64_t> axes;
-    Status status = info.GetAttrs<int64_t>("axes", axes);
-    ORT_ENFORCE(status.IsOK(), "Attribute axes is not set.");
+    // Parse attribute 'axes'
+    Status status = info.GetAttrs<int64_t>("axes", axes); 
+    // to avoid the unused variable warning
+    (void)(status);
 
-    // Handle out of order and repeating dims.
-    std::sort(axes.begin(), axes.end());
-    axes.erase(std::unique(axes.begin(), axes.end()), axes.end());
+    // Handle out of order and repeating dims for non-empty 'axes'.
+    if (axes.size() > 0) {
+      std::sort(axes.begin(), axes.end());
```

Comment:
May be slightly more direct/robust to use status instead of axes.size(), and only set axes_ if status is ok. If GetAttrs was to change in the future and fail half way through, axes could have some values that shouldn't be used.

Side note: ORT_IGNORE_RETURN_VALUE is mildly preferable to C-style cast to void

Reply:
Noted thanks.